modules/simulations/airflow-emreks/README.md

```diff
@@ -0,0 +1,100 @@
+## Introduction
+
+This module uses Airflow to trigger spark jobs on EMR on EKS. Also this module uses the `use-case` declared in this blogpost
+https://aws.amazon.com/blogs/big-data/manage-and-process-your-big-data-workflows-with-amazon-mwaa-and-amazon-emr-on-amazon-eks/
+
+For custom use-cases, you shuld be adjusting the required permissions on EMR Job execution role.
+
+## Inputs/Outputs
+
+
+### Input Parameters
+
+```yaml
+name: emr-on-eks
```

Comment:
Can you define the actual input and output parameters as well as add the examples....

Reply:
done